backend/docker-compose.yml

```
@@ -0,0 +1,76 @@
+      dockerfile: Dockerfile
+    command: uvicorn user_management.main:app --host 0.0.0.0 --port 8000 --reload
+    container_name: api
+    volumes:
+      - .:/app:cached  
+    ports:
+      - "8000:8000"
+    env_file:
+      - .env
+    environment:
+      DATABASE_URL: ${DATABASE_URL}
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)